forms/views.py

```diff
@@ -1,17 +1,26 @@
 from django.forms.models import model_to_dict
 from django.http import HttpRequest, HttpResponse, HttpResponseNotAllowed, JsonResponse, HttpResponseNotFound
 
-from forms.models import Form
+from .form import FormForms
+from .models import Form
 
 
 def form_view(request: HttpRequest) -> HttpResponse:
-    if request.method != "GET":
-        return HttpResponseNotAllowed("GET")
+    if request.method == "GET":
+        forms_query_set = Form.objects.all()
+        forms_json = list(forms_query_set.values())
+        return JsonResponse(forms_json, safe=False)
+
+    if request.method == "POST":
+        data = FormForms(request.POST)
+
+        if data.is_valid():
+            Form.objects.create(**data.cleaned_data)
+            return HttpResponse()
 
-    forms_query_set = Form.objects.all()
-    forms_json = list(forms_query_set.values())
+        return HttpResponse(status=500)
```

Comment:
Use `400` (Bad Request) for that -- 500 is for **unexpected** cases. Invalid input data from users is an **expected** case

Reply:
Done